### PR TITLE
Misused email translation

### PR DESCRIPTION
**Pull request by marharyta**
# Checklist for author

Remove items that aren't applicable, check items that are done.

## Reviewability
- [x] Link to issue (#2656 2434)
- [ ] Note if PR is on top of other PR
- [ ] Note if related change in rems-deploy repo
- [ ] Consider adding screenshots for ease of review

## Backwards compatibility
- [ ] API is backwards compatible or completely new
- [ ] Events are backwards compatible _or_ have migrations
- [ ] Config is backwards compatible

## Documentation
- [ ] Update changelog if necessary
- [ ] API is documented and shows up in Swagger UI
- [ ] Components are added to guide page
- [ ] Update docs/ (if applicable)
- [ ] Update manual/ (if applicable)
- [ ] ADR for major architectural decisions or experiments
- [ ] New config options in config-defaults.edn

## Testing
- [ ] Complex logic is unit tested
- [ ] Valuable features are integration / browser / acceptance tested automatically

## Follow-up
- [ ] New tasks are created for pending or remaining tasks

### CODE REVIEW

**Comment by marharyta** [comment]
@opqdonut I have looked through the UI, looks like this is the only case I see for now where we can replace the Email text

**Comment by marharyta** [comment]
It of coarse also depends on what policy we pick for names

**Comment by opqdonut** [comment]
Yeah I guess that's all there is.

**Comment by marharyta** [comment]
ping @joel and @Macroz should be all good

File: src/cljs/rems/profile.cljs [hunk 1]
```diff
@@ -100,7 +100,7 @@
        {:title (text :t.profile/your-details)
         :always [:<>
                  [user/username (:user identity)]
-                 [user/attributes (:user identity)]]}]]]))
+                 [user/attributes attributes false]]}]]]))
```
Comment (opqdonut) [inline src/cljs/rems/profile.cljs:103]: Passing booleans always looks so opaque, but in this case I don't think we want full keyword args just for one boolean... Nothing to fix here, just thinking out loud.

File: src/cljs/rems/user.cljs [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
 (defn attributes
   "A div with a rems.atoms/info-field for every user attribute in the given attributes."
```
Comment (opqdonut) [inline src/cljs/rems/user.cljs:17]: Please document the new parameter briefly

File: src/cljs/rems/user.cljs [hunk 3]
```diff
@@ -30,7 +30,10 @@
            (when-let [mail (:notification-email attributes)]
              [info-field (text :t.applicant-info/notification-email) mail {:inline? true}])
            (when-let [mail (:email attributes)]
-             [info-field (text :t.applicant-info/email) mail {:inline? true}])
+             [info-field (text :t.applicant-info/email) mail {:inline? true}]
```
Comment (opqdonut) [inline src/cljs/rems/user.cljs:33]: this line does nothing, right?

File: src/cljs/rems/user.cljs [hunk 4]
```diff
@@ -53,22 +56,33 @@
...
                          :email "developer@uu.id"
                          :name "Deve Loper"
                          :notification-email "notification@example.com"
                          :organizations [{:organization/id "Testers"} {:organization/id "Users"}]
                          :address "Testikatu 1, 00100 Helsinki"
                          :researcher-status-by "so"
-                         :nickname "The Dev"}])
-   (example "invalid value for researcher status"
+                         :nickname "The Dev"}
+             true])
```
Comment (opqdonut) [inline src/cljs/rems/user.cljs:78]: perhaps this case should only have `:email` and `:name` just like a realistic use case? not sure.

**Review by opqdonut — CHANGES_REQUESTED** [review]
good to merge after fixing the missing doc and the line that does nothing

File: src/cljs/rems/profile.cljs (hunk 1 above)
Comment (marharyta) [reply src/cljs/rems/profile.cljs:103]: I also thought so, but figured this is not that big of a deal.

File: src/cljs/rems/user.cljs (hunk 4 above)
Comment (marharyta) [reply src/cljs/rems/user.cljs:80]: Maybe, I probably can create another use case with "invited" user

**Review by opqdonut — APPROVED** [review]
:+1:

**Pull request merged**